_maps/configs/syndicate_gorlex_banshee.json

```diff
@@ -0,0 +1,43 @@
+{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"faction": "/datum/faction/syndicate",
+	"prefix": "ISV",
+	"map_name": "Y450-H Banshee-class Salvage Clipper",
+	"namelists": [
+		"GORLEX",
+		"NATURAL_AGGRESSIVE",
+		"BEASTS",
+		"WEAPONS"
+	],
+	"map_short_name": "Banshee-Class",
+	"description": "A drastic refit of an ICW-era Cybersun yacht into a salvage vessel. While the Banshee retains most of the original exterior, its interior houses an exosuit-based Hardliner salvage team and all the facilities they could need. Though somewhat dated, it functions well as a sleeper ship that could easily be mistaken for any other civilian vessel.",
+	"tags": [
+		"Salvage",
+		"Combat",
+		"Engineering",
+		"Generalist"
+	],
+	"map_path": "_maps/shuttles/syndicate/syndicate_gorlex_banshee.dmm",
```

Comment:
Hey, rename it to `syndicate_hardliner_banshee.dmm`. The "gorlex" part is outdated, and is being slowly removed on the NGR side of things.